### ISSUE TITLE
Editor not loading if page is set as "Posts page"

### ISSUE THREAD

**Issue by vgstef** [elementor/elementor#11236]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

If a page in set as the "Posts page" in Settings->Reading, we can't edit the page with Elementor. The editor starts loading, then stops. After a while, a popup appears to propose to "Activate safe mode", but it doesn't solve the issue.

## Steps to reproduce

1. Create a page with Wordpresse
2. Set this page for the "Posts page" field in Settings->Reading
3. Try to edit this page with Elementor.

If I remove the page in the "Posts page" field, then we can edit the page with Elementor.

Elementor should remove the page form the settings, or at least inform the user about the need to remove the page. 

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by lior-sh** [comment]
Hi @vgstef,

Thanks for reporting, we couldn't reproduce the issue.
Please follow guidelines for opening a new [issue report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md), so we can help you better.
This report is missing key information:
* Prerequisites.
* *Clear steps to replicate*.
* Isolating the problem.
* *System info*.
* One issue per thread
Without it, we simply can't help.
Thanks.

**Comment by vgstef** [comment]
Hi, thanks lior-sh for the follow-up,

I'll try to clarify the steps.

1. Create a page with Elementor called "Articles" and publish it.
2. Go to Settings-> Reading, than in the Pages post download list, chose the page "Articles"
3. Save settings and go to the pages sections and try to edit the page "Articles".

I tried on different serveur/installations with lastest updates, and it always do the same. The editor starts loading than an error message appears.

That's not a huge bug, but people should know that if a page is defined in the Settings -> Reading -> Posts page, they won't be able to edit that page with Elementor.

I had to search quite a but to find the problem.
